specification/postgresqlhsc/resource-manager/Microsoft.DBforPostgreSQL/preview/2020-10-05-privatepreview/examples/ServerGroupPromote.json

```diff
@@ -0,0 +1,12 @@
+{
+  "parameters": {
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "resourceGroupName": "TestGroup",
+    "serverGroupName": "hsctestsg1",
+    "api-version": "2020-10-05-privatepreview"
+  },
+  "responses": {
+    "200": {},
+    "202": {}
+  }
```

Comment:
what if it's an invalid request? Even if the server group is primary, should we accept the request?